### PR TITLE
docs: Add missed docstring to context; Hide _Context

### PR DESCRIPTION
**Pull request by Damego**
## About

This pull request is about (X), which does (Y).

## Checklist

- [ ] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
- [ ] I've ensured the change(s) work on `3.8.6` and higher.

I've made this pull request: (check all that apply)
  - [x] For the documentation
  - [ ] To add a new feature
  - [ ] As a general enhancement
  - [ ] As a refactor of the library/the library's code
  - [ ] To fix an existing bug
  - [ ] To resolve #ISSUENUMBER

This is:
  - [ ] A breaking change

### CODE REVIEW

File: interactions/client/context.py [hunk 1]
```diff
@@ -38,12 +38,6 @@ class _Context(ClientSerializerMixin):
     that the user-facing API is able to allow developers to
     easily access information presented from any event in
     a "contextualized" sense.
-
-    :ivar Optional[Message] message: The message data model.
-    :ivar Member author: The member data model.
-    :ivar User user: The user data model.
-    :ivar Optional[Channel] channel: The channel data model.
-    :ivar Optional[Guild] guild: The guild data model.
```
Comment (sillyElvy) [inline interactions/client/context.py:46]: keep it

**Review by sillyElvy — CHANGES_REQUESTED** [review]
see cmment

File: interactions/client/context.py [hunk 2]
```diff
@@ -600,10 +605,27 @@ class ComponentContext(_Context):
     """
     A derivation of :class:`_Context`
```
Comment (sillyElvy) [inline interactions/client/context.py]: remove link aso for commandctx but cant mention it

**Review by sillyElvy — CHANGES_REQUESTED** [review]

File: interactions/client/context.py (hunk 1 above)
Comment (Damego) [reply interactions/client/context.py:46]: I added these ivars to Command & Component Contexts.
Also _Context is hidden now so why we need to keep it?

Comment (sillyElvy) [reply interactions/client/context.py:46]: i mean users can see it but if you think its not needed leave it out

**Review by sillyElvy — APPROVED** [review]

**Pull request merged**